src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/MethodologyServiceCachingTests.cs

```diff
@@ -0,0 +1,646 @@
+                                    new MethodologyVersionSummaryViewModel
+                                    {
+                                        Id = Guid.NewGuid(),
+                                        Title = "Theme 1 Topic 1 Publication 2 Methodology 1",
+                                        Slug = "Theme 1 Topic 1 Publication 2 Methodology 1 Slug"
+                                    },
+                                    new MethodologyVersionSummaryViewModel
+                                    {
+                                        Id = Guid.NewGuid(),
+                                        Title = "Theme 1 Topic 1 Publication 2 Methodology 2",
+                                        Slug = "Theme 1 Topic 1 Publication 2 Methodology 2 Slug"
+                                    })
+                            })
+                    },
+                    new AllMethodologiesTopicViewModel
+                    {
+                        Title = "Theme 1 Topic 2",
+                        Publications = ListOf(
+                            new AllMethodologiesPublicationViewModel
+                            {
+                                Id = Guid.NewGuid(),
+                                Title = "Theme 1 Topic 2 Publication 1",
+                                Methodologies = ListOf(
+                                    new MethodologyVersionSummaryViewModel
+                                    {
+                                        Id = Guid.NewGuid(),
+                                        Title = "Theme 1 Topic 2 Publication 1 Methodology 1",
+                                        Slug = "Theme 1 Topic 2 Publication 1 Methodology 1 Slug"
+                                    }
+                                )
+                            })
+                    })
+            },
+            new AllMethodologiesThemeViewModel
+            {
+                Title = "Theme 2",
+                Topics = ListOf(
+                    new AllMethodologiesTopicViewModel
+                    {
+                        Title = "Theme 2 Topic 1",
+                        Publications = ListOf(
+                            new AllMethodologiesPublicationViewModel
+                            {
+                                Id = Guid.NewGuid(),
+                                Title = "Theme 2 Topic 1 Publication 1",
+                                Methodologies = ListOf(
+                                    new MethodologyVersionSummaryViewModel
+                                    {
+                                        Id = Guid.NewGuid(),
+                                        Title = "Theme 2 Topic 1 Publication 1 Methodology 1",
+                                        Slug = "Theme 2 Topic 1 Publication 1 Methodology 1 Slug"
+                                    })
+                            })
+                    })
+            }
+        );
```

Comment:
Do we need all of this test data? Feels like we could simplify this down and save ourselves a bunch of lines.

Reply:
I kind've wanted to have at least another theme and topic, another publication in the same theme and topic for this test case, just to be thorough and prove that we're just matching the correct data in this method.

Note that in other tests within this same test class, I'm not going overboard with them - but for this "happy path" one I just wanted to prove that only a very particular set of results is coming back.